docs/dao-contracts.md

```diff
@@ -0,0 +1,154 @@
+```mermaid
```

Comment:
Please, remove the Order part.
Also, you can update this diagram with the latest changes applied to the solidity smart contracts and merged this week.